build.ps1

```diff
@@ -230,8 +250,8 @@ function RunTests {
         $Filter
     )
 
-    $testAssemblyPath = "$solutionRoot/$Filter/bin/$Configuration/"
-    $testAssemblies = Get-ChildItem -Path $testAssemblyPath -Filter "$Filter.dll" -Recurse
+    $testAssemblyPath = "$solutionRoot/$Filter/"
+    $testAssemblies = Get-ChildItem -Path $testAssemblyPath -Filter "$Filter.csproj" -Recurse
```

Comment:
Coverlet code coverage reporting with "XPlat Code Coverage" (line 267) does not work on DLLs. Need to find the C# project instead.